customers/Dockerfile

```diff
@@ -21,7 +21,6 @@ RUN export CP_SHA1=198d96c8d7bfafb1ab6df96653c29701510b833c \
     && rm /tmp/containerpilot.tar.gz
 
 # add our application
-COPY containerpilot.json /opt/customers/
```

Comment:
This eliminates the file itself, not the environment variable. What are we trying to do here?